fzysearch/__init__.py

```diff
@@ -1,3 +1,37 @@
+# Input the two numbers of which the edit distance is to be calculated 
+def levenshtein_distance(word1,word2):
```

Comment:
use `first_word` or `first` and `second_word` or `second` instead, better names.